src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -2,27 +2,84 @@
 
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.junit.MatcherAssert.assertThat;
+import static org.joda.time.DateTimeConstants.APRIL;
 
 import java.net.MalformedURLException;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
+import org.folio.circulation.domain.policy.Period;
 import org.folio.circulation.support.http.client.IndividualResource;
 import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
 import org.junit.Test;
 
 import api.support.APITests;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.OverrideRenewalByBarcodeRequestBuilder;
+import api.support.builders.RenewByBarcodeRequestBuilder;
+import api.support.builders.RenewByIdRequestBuilder;
 import api.support.builders.RequestBuilder;
 import api.support.http.InventoryItemResource;
 
 public class RequestsAPILoanRenewalTests extends APITests {
+
+  @Test
+  public void forbidRenewalLoanByBarcode_firstRequestIsRecall() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
```

Comment:
I don't think we have used underscores in test names elsewhere, can you help me understand one has been used here?

Reply:
The main aim is to have after underscore a condition

Reply:
Ok, so it is to distinguish the area being tested, from the condition?

In method centric unit tests I can understand this kind of naming convention (though I tend to use classes for grouping).

In API tests, what comes before and after the underscore (as there is no method)?

I’d prefer our meaning structure was consistent, so if we were to adopt something like this, I’d want it to be standardised.

@mreno-EBSCO @mtraneis what do you think to changing our naming conventions for tests?

(I can share my personal thoughts on test names if people would like)

Reply:
I like the idea but I think it's overkill for API tests because these are more workflow/scenarios based.

Reply:
Ok, I will rename them

Reply:
Done